Address-Finder/Address-Finder/Views/AddressTableCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  TableClass.swift
+//  Address-Finder
+//
+//  Created by Daye on 2021/07/29.
+//
+
+import UIKit
+
+class AddressTableCell: UITableViewCell {
+  let placeNameLabel: UILabel = {
+    let label = UILabel()
+    label.font = UIFont.preferredFont(forTextStyle: .headline)
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+  let roadAddressLabel: UILabel = {
+    let label = UILabel()
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+  let jibeonAddressLabel: UILabel = {
+    let label = UILabel()
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
```

Comment:
```suggestion
  let placeNameLabel: UILabel = {
    let label = UILabel()
    label.font = UIFont.preferredFont(forTextStyle: .headline)
    label.translatesAutoresizingMaskIntoConstraints = false
    return label
  }()
  
  let roadAddressLabel: UILabel = {
    let label = UILabel()
    label.translatesAutoresizingMaskIntoConstraints = false
    return label
  }()
  
  let jibeonAddressLabel: UILabel = {
    let label = UILabel()
    label.translatesAutoresizingMaskIntoConstraints = false
    return label
  }()
```
추후부터는 UI 프로퍼티끼리 한줄 개행하도록 하겠습니다!!
개행도 의미있게 작성하여 코드에 다같이 녹여냈으면 좋겠습니다~